robottelo/ui/locators/base.py

```diff
@@ -1279,6 +1279,9 @@
     "rh.prd_expander": (
         By.XPATH, ("//div[@id='ui-tabs-1']//td[contains(.,'%s')]"
                    "/span[@class='expander']")),
+    "rh.ostree_prd_expander": (
+        By.XPATH, ("//div[@id='ui-tabs-7']//td[contains(.,'%s')]"
```

Comment:
```
//li/a[contains(@href, 'ostree')]
```
or
```
//li/a[contains(@href, 'tab=ostree')]
```
or
```
//li/a[text()='OSTree']
```
Using of text is not a preferable option, so probably stick to href. In that way you will be safe from change of tab order and make your code more specific